.github/workflows/macos.yml

```diff
@@ -71,3 +71,32 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
+
+  uniffi-check:
+    # macos-*-xlarge is needed to get M1 chips - https://github.blog/2023-10-02-introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/
```

Comment:
Because we are using `self-hosted` and not the GitHub hosted one, the original comment this was copied from can be removed and this copied comment too.